index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+    
+</head>
+<body>
+    <section class="coverpage">
+    <header class="topheader">
+        <a href="#" class="logo">RK</a>
+        <nav class="menu">
+            <a href="#" class="togglenav">
+                <img src="images/Union.png" alt="menu_button">
+            </a>
+        </nav>
+    </header>
+    <br>
+    <br>
+    <section class="backgroundimg">
+        <div class="personal-info">
+            <h1 class="introname">I'm Richard Karanu <br> Glad to see <br> you!</h1>
+            <p class="intro">
+                I'm a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what
+                you see and have a project you need coded, don't hestiate to contact
+                me.
+            </p>
+            <h6 class="connect">LET'S CONNECT</h6>
```

Comment:
- I think it will be better if we wrapped our ` LET'S CONNECT` in a `<button>` tag or an `<a>` tag as this defines a clickable link or will let you connect to other sections of the website using `href`. 

We can refactor our code to look something like this 👉🏽 `<h6 class="connect"><a href="">LET'S CONNECT</a></h6>`